astroquery/simbad/core.py

```diff
@@ -309,10 +309,13 @@ def list_votable_fields(self):
 
         if not hasattr(self,'_votable_fields_table'):
             # load the table
-            votable_fields_table = Table.read(
-                get_pkg_data_filename(os.path.join('data',
-                                                   'votable_fields_table.txt')),
-                format='ascii')
+            dict_file = get_pkg_data_filename(
+                os.path.join('data', 'votable_fields_dict.json'))
+            with open(dict_file, "r") as f:
+                fields_dict = json.load(f)
+            rows_to_add = list(zip(*[iter(sorted(list(fields_dict.keys())))]))
```

Comment:
I believe the Table was being used for formatting purposes; there are a few too many possible fields to print readably in a single list.  So, if there's a way to print the result so that it shows up nicely on a "standard" 100-line, 80-wide screen (I don't know if that's the correct standard), it would be best.